<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LearnStartDialog</class>
 <widget class="QDialog" name="LearnStartDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>    开始学习   </string>
  </property>
  <property name="windowIcon">
   <iconset resource="rc.qrc">
    <normaloff>:/new/prefix2/rc/Wordpress.png</normaloff>:/new/prefix2/rc/Wordpress.png</iconset>
  </property>
  <widget class="QGroupBox" name="InfoGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>图形信息</string>
   </property>
   <widget class="QSint::PieChart" name="PieChart" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>291</width>
      <height>241</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="InfoListWidget">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="flow">
     <enum>QListView::TopToBottom</enum>
    </property>
    <property name="isWrapping" stdset="0">
     <bool>false</bool>
    </property>
    <property name="resizeMode">
     <enum>QListView::Adjust</enum>
    </property>
    <property name="layoutMode">
     <enum>QListView::SinglePass</enum>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="uniformItemSizes">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="selectionRectVisible">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>未学习</string>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
     <property name="background">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </item>
    <item>
     <property name="text">
      <string>已学习</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
     <property name="background">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>255</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="InfoLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>学习进度饼状图</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="LearnGroupBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>271</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>学习选择</string>
   </property>
   <widget class="QPushButton" name="LearnPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始学习</string>
    </property>
    <property name="icon">
     <iconset resource="rc.qrc">
      <normaloff>:/new/prefix1/rc/actions/start.png</normaloff>:/new/prefix1/rc/actions/start.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="ReviewPushButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始复习</string>
    </property>
    <property name="icon">
     <iconset resource="rc.qrc">
      <normaloff>:/new/prefix1/rc/actions/player_time.png</normaloff>:/new/prefix1/rc/actions/player_time.png</iconset>
    </property>
   </widget>
   <widget class="QGroupBox" name="ChoiceGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>选择</string>
    </property>
    <widget class="QCheckBox" name="ChoiceCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>按单元学习复习</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="LekLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>51</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>选择单元：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="LektionComboBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="modelColumn">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="LektionLineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>输入单元：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ReviewLektionComboBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="InfoTextBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <property name="isWrapping" stdset="0">
     <bool>false</bool>
    </property>
    <property name="uniformItemSizes">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>ChoiceGroupBox</zorder>
   <zorder>LearnPushButton</zorder>
   <zorder>ReviewPushButton</zorder>
   <zorder>InfoTextBrowser</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSint::PieChart</class>
   <extends>QWidget</extends>
   <header>QSintCharts</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
